data/the emerald sword.txt

```diff
@@ -0,0 +1,803 @@
+			`	"Ten million credits." They say. "Non negotiable. And don't try anything funny."`
+			branch cash
+				credits >= 12000000
+			choice
+				`	"What? I don't have that much money"`
+					goto nomoney
+				`	"That's expensive. If it's such a trivial service, can't I have a discount?"`
+					goto haggle
+				`	"How about you tell me what I want and I don't blow up your ship and your pirate friends?"`
+					goto threat
+			label visited
+			`You come back to Stormhold, And receive new coordinates to meet with Mammon. Would you like to visit them?`
+			choice
+				`	(Yes)`
+				`	(No)`
+					defer
+			`	You follow the new coordinates, arriving again in a random place in the sea, again under stormy conditions, which seems to be the norm in this planet. The submarine emerges, you land, and follow the men to Mammon's room.`
+			`	"Welcome back." They say.`
+			label question
+			`	"I hope you are ready to make your payment."`
+			branch cash
+				credits >= 12000000
+			choice
+				`	"I don't have that much money to spare."`
+					goto nomoney
+				`	"Are you ready to give me a discount?"`
+					goto haggle
+				`	"How about you tell me what I want and I don't blow up your ship and your pirate friends?"`
+					goto threat
+			label cash
+			choice
+				`	"I need to think about this. Give me some time."`
+					defer
+				`	"That's a lot of money just for information. Can you make it cheaper?"`
+					goto haggle
+				`	"How about you tell me what I want and I don't blow up your ship and your pirate friends?"`
+					goto threat
+				`	"Here's the money."`
+					goto pay
+			label nomoney
+			`	"I know you don't. I can arrange for the Payment to be made in different ways. Organ donation, prostitution, voluntarizing as a test subject... We'll need an appraisal first though."`
+			`	You politely decline.`
+			`	"Come back to me when you have it, then, and I shall provide you with the information. We have all the time in the universe. Until you die, or someone decides to destroy the cube, that is."`
+			`	They gesture, and the men promptly and diligently escort you back out to your ship.`
+				defer
+			label haggle
+			`	Mammon doesn't look pleased.`
+			`	"What part of non-negotiable did you not understand? You're pushing your luck, captain <last>."`
+				goto question
+			label threat
+			`	Mammon just smiles, not at all intimidated.`
+			`	"You think you can intimidate me? It's your funeral, human. If you were smart, you would know that my power far exceeds yours. You know I can't let you leave here alive now. It would set a bad precedent for me. Safety is my top priority."`
+			`	You look around, and the men in suit are pointing guns at you. Alien guns. Before you can react, Mammon shoots you from under the table, fatally injuring your abdomen. Your half conscious body is dragged out of the ship, and your fleet being blown out of the sky by alien weapons is the last thing you see.`
```

Comment:
This text assumes the player has a fleet at this point, but the player might just be flying alone or might have lost his/her fleet in the previous battle. Maybe change to "your ship being shredded to pieces by alien weapons is the last thing you see"